packages/cadl-ranch-dashboard/src/components/dashboard-table.tsx

```diff
@@ -286,5 +292,19 @@ function createTree(manifest: ScenarioManifest): ManifestTreeNode {
 
     current.scenario = scenario;
   }
-  return root;
+
+  return cutTillMultipleChildren(root);
```

Comment:
Remove unnecessary parent nodes which only contain 1 child.

before:
- Azure:
  - ResourceManger
    - Models
      - CommonTypes
      - Resources

Now: 
- CommonTypes
- Resources

Reply:
Or we could expand root nodes with 1 child by default, to better reflect the actual structure.

Reply:
I think ideally we would only collapse into a single node(like vscode does) but I understand the need here. Maybe what you want instead is to say you want to only show a subgraph(Azure.AzureResourceManager) instead of filtering

Reply:
Not exactly, but I think I get your point. 
Do you mean instead of having 3 separate graphs, we could have one graph be the subgraph of a total group of scenarios, projected by each mode? I do believe it's better.